### PR TITLE
Expose a sample Genesis Block from Framework

### LINKED ISSUE #3308: Expose a sample Genesis Block from Framework
### Expected behavior

SDK user must be able to start the blockchain application with the minimum code required. 

```js
const {Application, sampleGenesisBlock } = require('lisk-framework');

const app = new Application('myApp', sampleGenesisBlock);

app.run().then(console.info).catch(console.error);
```

To make this possible we should expose one genesis block as part of the framework, which everyone can start with easily. 

### Actual behavior

Right now the genesis block is the required data field to start any application. And we are not shipping any sample genesis block for user. So he had to go to one extra step to first create a genesis block to start any application. 

### Which version(s) does this affect? (Environment, OS, etc...)
1.7

### PR DESCRIPTION
**Pull request by MaciejBaj**
### What was the problem?
For the development of Alpha SDK purposes, Framework should expose a genesis block. 

### How to test it?
Check if `const { GenesisBlockDevnet } = require('lisk-framework');`  works

### Review checklist

* The PR resolves #3308 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: framework/src/index.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 const Application = require('./controller/application');
+const GenesisBlockDevnet = require('./modules/chain/defaults/genesis_block_devnet');
```
Comment (nazarhussain) [inline framework/src/index.js]: Its an object not a constructor function so we use camel case `genesis*`

Comment (nazarhussain) [inline framework/src/index.js]: As of now we have no reference or usage of concept `network` int he framework. Network is just the part of the Lisk Core implementation. So I suggest we simply rename it to `sampleGenesisBlock` and don't opt Network naming convention in the framework (at least in current state).

File: framework/src/modules/chain/defaults/genesis_block_devnet.json [hunk 2]
```diff
@@ -0,0 +1,1760 @@
+{
```
Comment (nazarhussain) [inline framework/src/modules/chain/defaults/genesis_block_devnet.json]: As of the separation of concern, the genesis block concept in current implementation is part of the application/controller layer, and passed to every module as static data option. 

So we should keep that file to application level.

Comment (nazarhussain) [inline framework/src/modules/chain/defaults/genesis_block_devnet.json]: Shouldn't we have a different genesis block been part of framework? Or we use devnet genesis block which comes from Lisk Core our network implementation.

File: framework/src/index.js (hunk 1 above)
Comment (MaciejBaj) [reply framework/src/index.js]: Done.

Comment (MaciejBaj) [reply framework/src/index.js]: I moved `genesis_block_devnet` to `framework/samples` directory I exported all the samples from there. 

It is a Genesis Block for development purposes, so `devnet` genesis block is correct to me.

File: framework/src/modules/chain/defaults/genesis_block_devnet.json (hunk 2 above)
Comment (MaciejBaj) [reply framework/src/modules/chain/defaults/genesis_block_devnet.json]: It is a Genesis Block for development purposes, so devnet genesis block is correct to me.

File: README.md [hunk 3]
```diff
@@ -66,9 +66,9 @@ To create a blockchain application, you need to provide an entry point of your a
 It is quite simple to have a working blockchain application, mirroring the configuration of the existing Lisk network. This can be done by copying the following three lines of code to your `index.js`:
 
 ```js
-const { Application, SampleGenesisBlock } = require('lisk-framework');
+const { Application, genesis_block_devnet } = require('lisk-framework');
```
Comment (nazarhussain) [inline README.md]:
```suggestion
const { Application, genesisBlockDevnet } = require('lisk-framework');
```

File: README.md [hunk 4]
```diff
@@ -66,9 +66,9 @@ To create a blockchain application, you need to provide an entry point of your a
...
 
-const app = new Application('my-app', SampleGenesisBlock);
+const app = new Application('my-app', genesis_block_devnet);
```
Comment (nazarhussain) [inline README.md]:
```suggestion
const app = new Application('my-app', genesisBlockDevnet);
```

File: framework/samples/index.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+const genesis_block_devnet = require('./genesis_block_devnet');
```
Comment (nazarhussain) [inline framework/samples/index.js]:
```suggestion
const genesisBlockDevnet = require('./genesis_block_devnet');
```

File: framework/samples/index.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+
+module.exports = {
+	genesis_block_devnet,
```
Comment (nazarhussain) [inline framework/samples/index.js]:
```suggestion
	genesisBlockDevnet,
```

File: framework/src/index.js [hunk 7]
```diff
@@ -9,6 +10,7 @@ const validator = require('./controller/helpers/validator');
 module.exports = {
 	Application,
 	version,
+	...samples,
```
Comment (nazarhussain) [inline framework/src/index.js]: Would be nice if we keep the sample namespace, may be we have to export more sample data/fixtures in future as well as part of framework.

**Pull request merged**